addons/account_check_printing/models/account_payment.py

```diff
@@ -263,3 +263,9 @@ def _check_make_stub_line(self, invoice):
             'amount_paid': formatLang(self.env, invoice_sign * amount_paid, currency_obj=self.currency_id),
             'currency': invoice.currency_id,
         }
+
+    def _auto_init(self):
```

Comment:
Should we use `def _auto_init` or `def init`?

I ran the following rgrep command to detect the most common method used:
 - `rgrep "tools.create_index" . --include=*.py -B3 |grep def`

And it is `def _auto_init` but it is private (_) and the docstring looks like best option `def init`
For that, I'm confused what should we use here